main.js

```diff
@@ -0,0 +1,38 @@
+var inputDishes = document.querySelectorAll("input[name='dish']");
+var btnLetsCook = document.querySelector(".lets-cook-btn");
+var imgCookPot = document.querySelector("svg");
+var randomDishIntro = document.querySelector("p");
+var randomDishText = document.querySelector(".random-dish");
+var randomDish;
+
+btnLetsCook.addEventListener("click", displayRandomDish);
+
+//retrieves random array index
+function randomIndex(array) {
```

Comment:
avoid variables that are named after the data type i.e. "array", "bool", "char", etc

Reply:
Will change this thanks!